### PR TITLE
feat: Remove rebuild index of separated indexer

### PR DESCRIPTION
**Pull request by yanguoyu**
Refer to https://github.com/Magickbase/neuron-public-issues/issues/330

https://github.com/nervosnetwork/neuron/assets/12881040/cfac896f-6995-45d5-9a80-a6af56c93772

### CODE REVIEW

File: packages/neuron-ui/src/locales/en.json [hunk 1]
```diff
@@ -411,28 +411,13 @@
...
-        }
-      },
       "data": {
         "ckb-node-data": "CKB Node Config & Storage",
         "set": "Set",
         "cache": "Cache",
         "clear-cache-description": "Clear cache if you encounter data sync or balance display problems. Neuron will rescan block data.",
-        "cache-cleared-on": "Cache cleared on {{date}}",
-        "clear-cache": "Clear Cache",
+        "cache-cleared-on": "Refresh cache on {{date}}",
```
Comment (Keith-CY) [inline packages/neuron-ui/src/locales/en.json:419]:
```suggestion
        "cache-cleared-on": "Refreshed cache on {{date}}",
```

**Review by Keith-CY — APPROVED** [review]

**Comment by homura** [comment]
It would be helpful if the description provided some context or reason for this PR

**Comment by Keith-CY** [comment]
> It would be helpful if the description provided some context or reason for this PR

cc @yanguoyu

**Comment by yanguoyu** [comment]
> > It would be helpful if the description provided some context or reason for this PR
> 
> cc @yanguoyu

I have added the issue that refers to this PR at https://github.com/nervosnetwork/neuron/pull/2990#issue-2042719782

**Comment by Keith-CY** [comment]
> > > It would be helpful if the description provided some context or reason for this PR
> > 
> > 
> > cc @yanguoyu
> 
> I have added the issue that refers to this PR at [#2990 (comment)](https://github.com/nervosnetwork/neuron/pull/2990#issue-2042719782)

I see, please review again @homura

**Review by homura — APPROVED** [review]
PR is LGTM, and I'd suggest that the tile could be changed to "remove rebuild index of separated indexer"

**Review by silySuper — APPROVED** [review]

**Pull request merged**